test/service/test_async_manifest_service.py

```diff
@@ -206,6 +223,8 @@ def test_manifest_endpoint_start_execution(self, mock_uuid, step_function_helper
                     self.assertEqual(301, response['Status'] if fetch else response.status_code)
                     self.assertIn('Retry-After', response if fetch else response.headers)
                     self.assertIn('Location', response if fetch else response.headers)
+                    if fetch:
+                        self.assertIn('CommandLine', response)
```

Comment:
Surely this can be made stronger.

Reply:
This addition was made to match the existing checks in this test case that only verify the existence of the keys `Retry-After` and `Location` in the response. The value of the `CommandLine` key is verified in `test_manifest.py`'s `test_fetch_manifest()`

Reply:
Sounds good.

Reply:
Might make sense to explain that in a very short comment.